dreamjub/management/commands/ldapsync.py

```diff
@@ -1,22 +1,24 @@
-from django.core.management.base import BaseCommand, CommandError
-from jDREAM.models import Student
+from django.core.management import base
+from dreamjub.models import Student
```

Comment:
Module level imports only. This can be redone as `import dreamjub.models as dreamjub_models` and then the callable class can be fetched as `dreamjub_models.Student`